icu4c/source/common/unicode/uconfig.h

```diff
@@ -323,6 +323,16 @@
 #   define UCONFIG_NO_NORMALIZATION 0
 #endif
 
+/**
```

Comment:
@markusicu This looks like an API change to me (because the file is located in common/unicode directory) , need to go through ICU TC to discuss this, right?

Reply:
We should initially start with marking this one `@internal`.

Reply:
Done